General Workflow Upgrades and Separation of Sampler/Estimator Primitives using REST API notebooks

born-2learn opened the pull request:
Opening this PR to propose general workflow upgrades. This PR is a work in progress.

- [x] Added Sampler and Estimator notebooks.

johannesgreiner commented:
@born-2learn can you specify details about the two different execution modes in here? Atm I find batch and session interleaved & not explicitly defined. In the "Create Session" session, consider to leave out h/g/p to be consistent with above, or convert from variable names for hub/group/project above

born-2learn commented:
Thanks for the feedback @johannesgreiner .

born-2learn commented:
@johannesgreiner the latest commit proposes the following changes:
1. Batch/Session modes are separated with explanations
2. Added links to IBM Quantum Documentation guides pertaining to REST APIs
3. Fixed h/g/p at all locations to stay consistent

johannesgreiner commented:
LGTM, thank you @born-2learn .
